tasks/managed/collect-cosign-params/README.md

```diff
@@ -4,9 +4,19 @@ Tekton task that collects cosign options from the data file.
 
 ## Parameters
 
-| Name      | Description                                                             | Optional | Default value  |
-|-----------|-------------------------------------------------------------------------|----------|----------------|
-| dataPath  | Path to the JSON string of the merged data to use in the data workspace | No       | -              |
+| Name                    | Description                                                                                                                | Optional | Default value                                             |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------|----------|-----------------------------------------------------------|
```

Comment:
missing trustedArtifactsDebug and orasOptions

Reply:
fixed